tests/bgp_commands_test.py

```diff
@@ -25,32 +25,32 @@
 Peer groups 4, using 256 bytes of memory
 
 
-Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down    State/PfxRcd    NeighborName
```

Comment:
Great find, but could you please do global search 'Neighbhor' in sonic-mgmt to see whether we already uses the wrong magic string?
If so, let's do compatible support first in sonic-mgmt for the correction

Reply:
Good idea! I know a few places are using 'Neighbhor' in sonic-mgmt. Will raise PR to fix them as well.